app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -86,6 +91,7 @@
     private static final int ANIMATION_DURATION = 300;
     private static final String ARGUMENT_URL = "url";
     private static final String RESTORE_KEY_DOWNLOAD = "download";
+    private static final String SCREENSHOT_PATH = Environment.getExternalStorageDirectory() + File.separator + Environment.DIRECTORY_PICTURES + File.separator + "Screenshots" + File.separator;
```

Comment:
Samsung is using a different directory for screenshots, we should either watch a set of directories or try to guess where the screenshot directory is with the device info.

Reply:
Let's start with listening to the root and on every event check if the path has a "screenshot" in it (case insensitive, does not check for /screenshot/ since maybe a folder could have the name screenshots screenshot_ ... etc to cover most of the cases. Let's ignore if there are some devices that have a different name for screenshot folders for now. We'll file another bug if such device exist.) Listening to the root may raise some performance issue but let's start with the easiest one and see if it works. 😃